DEVELOPMENT.md

```diff
@@ -162,28 +158,41 @@ Tests are written as advised in the [Terraform docs](https://developer.hashicorp
 
 To run the tests:
 1. Ensure MAAS_API_KEY and MAAS_API_URL environment variables are set (see [Running the local provider](#running-the-local-provider)).
-1. Run the unit tests with:
-    ```bash
-    make test
-    ```
-1. Run both the unit tests and terraform acceptance tests with:
-    ```bash
-    make testacc
-    ```
-    Note that you may need to specify a specific environment variables for some tests to pass, for example machine ids. Add these to your `env.sh` file before sourcing it again, if required:
-    ```bash
-    export TF_ACC=1
-    export TF_ACC_NETWORK_INTERFACE_MACHINE=<machine_id> # e.g. b68rn4
-    export TF_ACC_TAG_MACHINES=<machine_id> # e.g. b68rn4
-    ```
+
+2. Run the tests:    
+    - Run the unit tests:
+        ```bash
+        make test
+        ```
+    - Run both the unit tests and all terraform acceptance tests:
+        ```bash
+        make testacc
+        ```
+        > [!NOTE]
+        > You may need to specify specific environment variables for some tests to pass, for example machine ids. Add these to your `env.sh` file before sourcing it again, if required:
```

Comment:
`specify specific` -> `specify certain` / `set specific` ?

the doubling up right next to each-other reads a little strangely